docs/yaml/generate.go

```diff
@@ -17,12 +17,12 @@ import (
 const descriptionSourcePath = "docs/reference/commandline/"
 
 func generateCliYaml(opts *options) error {
-	dockerCli, err := command.NewDockerCli()
+	dockerCLI, err := command.NewDockerCli()
```

Comment:
Shouldn't we do this rename to the entire code then? I think I mostly used `dockerCli` 😅

Reply:
yes, Ideally we should; likely that would break some people though, so that's why I didn't do it as part of this PR 😊

Reply:
@thaJeztah oh you mean we would have to change "NewDockerCli" to "NewDockerCLI"? I don't think it's worth changing these little things but consistency is more important. So I'd vote for keeping it as-is.